examples/src/main/scala/tofu/example/doobie/TofuDoobieExample.scala

```diff
@@ -0,0 +1,142 @@
+package tofu.example.doobie
+
+import cats.data.ReaderT
+import cats.effect.{ContextShift, Effect, IO, IOApp, Sync}
+import cats.{Apply, Monad}
+import cats.tagless.syntax.functorK._
+import derevo.derive
+import doobie._
+import doobie.implicits._
+import doobie.util.log.LogHandler
+import tofu.doobie.log.{EmbeddableLogHandler, LogHandlerF}
+import tofu.doobie.transactor.Txr
+import tofu.doobie.LiftConnectionIO
+import tofu.higherKind.derived.representableK
+import tofu.lift.{Lift, UnliftIO}
+import tofu.logging.derivation.{loggable, loggingMidTry}
+import tofu.logging.{Logging, LoggingCompanion, Logs}
+import tofu.syntax.context._
+import tofu.syntax.monadic._
+import tofu.syntax.doobie.log.handler._
+import tofu.syntax.doobie.log.string._
```

Comment:
`import tofu.syntax.doobie.log._`
This looks like more unified way to import syntax

Reply:
These 2 imports should be usually put in different files.